### PR TITLE
Silk Touch Enchantments

### PR DESCRIPTION
**Pull request by aPunch**
This re-adds all of the silk touch effects that were reverted before.

### CODE REVIEW

File: src/main/java/org/spout/vanilla/material/block/controlled/SilverfishStone.java [hunk 1]
```diff
@@ -24,19 +24,19 @@
...
 	public SilverfishStone(String name, int id) {
 		super(name, id);
-		this.setController(VanillaControllerTypes.SILVERFISH_STONE);
 	}
 
 	@Override
 	public void initialize() {
 		super.initialize();
-		this.setHardness(0.8F).setResistance(1.2F); //TODO Confirm that this is the correct value
+		this.setHardness(0.8F).setResistance(1.2F).setController(VanillaControllerTypes.SILVERFISH_STONE); // TODO Confirm that this is the correct value
```
Comment (Zidane) [inline src/main/java/org/spout/vanilla/material/block/controlled/SilverfishStone.java]: Should be in the constructor I believe (setController).

Comment (aPunch) [inline src/main/java/org/spout/vanilla/material/block/controlled/SilverfishStone.java]: Well, there were some setController's in the constructor and some in the init method, so I moved them all to the init for consistency. I'll move them to the constructor, now.

EDIT - Done.

**Comment by windy1** [comment]
The getDrops method has a lot of duplicate code, we can probably abstract at least some of it, no?

**Comment by aPunch** [comment]
@W1ndwaker Not anymore than what was in the getDrops before. There are so many use-cases that creating more helper methods in EnchantmentUtil would seem like a waste unless everything was _really_ generalized. And even then I would think there would be duplicated code. I'll see what I can do to cut down on it, though.

**Comment by windy1** [comment]
@aPunch Yeah I know it wasn't you just stating it, if you could do something about it that would've awesome :)

**Comment by LunNova** [comment]
@W1ndwaker 

I'm working on that.

**Pull request merged**